tox.ini

```diff
@@ -25,12 +24,18 @@ setenv =
     LOCKDIR = {toxinidir}{/}requirements{/}locks
     TMPFILE = {envtmpdir}{/}geovista.yml
     YMLFILE = {toxinidir}{/}requirements{/}geovista.yml
+    ENVNAME = tmp-locks
 skip_install =
     true
 commands =
     cp {env:YMLFILE} {env:TMPFILE}
     python -c 'from sys import version_info as v; fh = open("{env:TMPFILE}", "a"); fh.write(f"\n  - python =\{v.major\}.\{v.minor\}\n")'
-    time conda-lock --channel conda-forge --kind explicit --file {env:TMPFILE} --platform linux-64 --filename-template "{envname}-\{platform\}.txt" {posargs}
+    time conda env create --name {env:ENVNAME} --file {env:TMPFILE} {posargs}
+    python -c \
+    'from subprocess import run;\
+     output_file = open("{envname}-linux-64.txt", "w");\
```

Comment:
Forgot to mention: I did successfully experiment with inheriting the platform name from the tox `platform` setting, then sanitising it using `re.sub()` (the current setting of `linux|darwin` made for unsafe file names). But the current setup looks deliberately locked down to `linux-64` so that's what I went for.